### PR TITLE
chore(tools): use bash instead of sh for pushd and popd

### PR DESCRIPTION
**Pull request by tbushman**
#### Pre-Submission Checklist

- [ ] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file
- [ ] Working changes have been added to the build by running `npm run build`

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.

- [ ] Closes currently open issue: Closes #XXX

### CODE REVIEW

**Comment by tbushman** [comment]
I need to merge this in order to test it. This uses `after_success` to ensure cwd of $TRAVIS_BUILD_DIR and change permissions on ./deploy.sh , then uses `bash` instead of `sh` in the deploy process

File: .travis.yml [hunk 1]
```diff
@@ -43,9 +43,12 @@ after_failure: |
     git push -f -q https://$GITHUB_USER:$GITHUB_API_KEY@github.com/$TRAVIS_REPO_SLUG travis-screenshots 2>/dev/null
     cd "$TRAVIS_BUILD_DIR"
   fi
+after_success:
+  - cd $TRAVIS_BUILD_DIR
+  - chmod +x ./deploy.sh
```
Comment (ValeraS) [inline .travis.yml]: @tbushman it is not needed if bash is used before the script.

File: .travis.yml [hunk 2]
```diff
@@ -43,9 +43,12 @@ after_failure: |
...
 deploy:
   provider: script
   skip_cleanup: true
-  script: ./deploy.sh
+  script: bash ./deploy.sh
   on:
```
Comment (ValeraS) [inline .travis.yml]: @tbushman `./` is not needed

**Comment by tbushman** [comment]
`@tbushman calm down, please.`  
. . . I am researching resolutions. I shared links in another discussion. What do you know about my personal disposition? 
The current status is `Deploying Application`  
No errors. Also, no build file at fCC/CDN.   
> ./ is not needed  
Alright, I'll try that with my usual zen calm.

**Comment by ValeraS** [comment]
@tbushman Excuse me. I don't want to hurt you. My english not good enough, and I can say something wrong.

it commits into CDN repo only if files in `/src` folder was changed

**Comment by tbushman** [comment]
Well, then. It appears to be working.

**Pull request merged**